deeppavlov/core/agent_v2/postprocessor.py

```diff
@@ -0,0 +1,37 @@
+from typing import Sequence
+
+from deeppavlov.models.preprocessors.person_normalizer import PersonNormalizer
+from deeppavlov.models.tokenizers.utils import detokenize
+
+
+class Postprocessor:
```

Comment:
```suggestion
class Postprocessor(metaclass=ABCMeta):
```

Reply:
I don't think Postprocessor should be an abstract class. The basic Postprocessor should be implemented the same way as basic *Preprocessor*, basic *ResponseSelector* and other basic classes. All these basic classes have `rest_caller` param in `__init__()` and not implemented `__call__()`. The idea behind `rest_caller` is that all selectors are separate services. So this postprocessor probably should even have a different name, because it is not 'default'.
And maybe we should even create a single basic class for all these classes.